_topic_maps/_topic_map.yml

```diff
@@ -3222,13 +3222,13 @@ Topics:
   - Name: Virtual machine networking
     Dir: vm_networking
     Topics:
-    - Name: Using the default pod network with OpenShift Virtualization
+    - Name: Configuring the default pod network with OpenShift Virtualization
       File: virt-using-the-default-pod-network-with-virt
       Distros: openshift-enterprise
-    - Name: Using the default pod network with OKD Virtualization
+    - Name: Configuring the default pod network with OKD Virtualization
       File: virt-using-the-default-pod-network-with-virt
       Distros: openshift-origin
-    - Name: Attaching a virtual machine to multiple networks
+    - Name: Configuring a virtual machine for a Linux bridge network
```

Comment:
ditto

Reply:
Not sure I agree with @phoracek . 

The section includes configuring the VM for the bridge network. However, it also includes creating a node network policy and a network attachment definition.

I'm not sure I can think of a better title because it has to cover a couple procedures for configuring the Linux bridge and then configuring the VM to connect to the Linux bridge. We need to think about this.

Reply:
Leaving this open for consideration @apinnick.

Reply:
How about "Attaching a virtual machine to a Linux bridge network"? That might be a good compromise because it would cover the steps involved without getting too specific.

@phoracek Is this section header wording OK with you?

Reply:
That sounds good to me

Reply:
Done.